spec/models/user_spec.rb

```diff
@@ -1,5 +1,48 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject do
+    User.new(name: 'Alex', photo: '', bio: '', post_count: 0)
+  end
+  before { subject.save! }
+
+  describe 'Test validations are correct' do
+    it 'test the user is invalid without a name' do
+      expect(subject).to be_valid
+      subject.name = nil
+      expect(subject).to_not be_valid
+      subject.name = ''
+      expect(subject).to_not be_valid
+    end
+    it 'test the user is invalid without post_count' do
+      expect(subject).to be_valid
+      subject.post_count = nil
+      expect(subject).to_not be_valid
+    end
+  end
+
+  it 'test that the user is created properly' do
+    name = 'Alfred'
+    photo = 'https://images.unsplash.com/photo-1642024728966-7492bae9bb60?w=300'
+    # email = 'alfred@test.com'
+    # password = 'alfred123'
```

Comment:
- [ ] It will be nice to remove commented-out codes from your codebase. They make the codebase to be unreadable and untidy. Use git history if you must view previous codes.